nerfstudio/scripts/datasets/process_project_aria.py

```diff
@@ -184,59 +292,126 @@ class ProcessProjectAria:
     """Path to Project Aria Machine Perception Services (MPS) attachments."""
     output_dir: Path
     """Path to the output directory."""
+    points_file: Optional[Path] = None
+    """Path to the point cloud file (usually called semidense_points.csv.gz) if not in the mps_data_dir"""
+    include_side_cameras: bool = False
+    """If True, include and process the images captured by the grayscale side cameras. If False, only uses the main RGB camera's data."""
+    vrs_file2: Optional[Path] = None
+    """Path to the second VRS file if provided"""
+    mps_data_dir2: Optional[Path] = None
+    """Path to the second MPS attachments if provided"""
+    points_file2: Optional[Path] = None
```

Comment:
it feels a little weird to me to hardcode things to only support 1 or 2 VRS files, instead of being able to do multiple

would it make sense to replace the original `vrs_file: Path` with `vrs_file: Tuple[Path, ...]`? as well as `mps_data_dir` with `mps_data_dir: Tuple[Path, ...]`?